rgw/admin/info_test.go

```diff
@@ -0,0 +1,24 @@
+//go:build ceph_preview
+// +build ceph_preview
+
+package admin
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func (suite *RadosGWTestSuite) TestGetInfo() {
+	suite.SetupConnection()
+	co, err := New(suite.endpoint, suite.accessKey, suite.secretKey, newDebugHTTPClient(http.DefaultClient))
+	assert.NoError(suite.T(), err)
+
+	suite.T().Run("test get rgw cluster/endpoint information", func(t *testing.T) {
+		info, err := co.GetInfo(context.Background())
+		assert.NoError(suite.T(), err)
+		assert.Equal(suite.T(), "rados", info.InfoSpec.StorageBackends[0].Name)
```

Comment:
Please add test case for cluster-id as well

Reply:
Done.